frames/section.lua

```diff
@@ -222,6 +234,31 @@ local function onTitleClickOrDrop(section)
   events:SendMessage('bags/FullRefreshAll')
 end
 
+---@param section Section
+local function onTitleRightClick(section) 
+  if section.headerDisabled then
+    return
+  end
+  
+  if (addon.atBank == false) then return end
+  
+  -- CellData[] { bagid, slotid, itemInfo, etc }
+  local items = {}
+
+  for _, cell in pairs(section.content.cells) do
+    if not cell.data.isItemEmpty then
+      table.insert(items, cell.data)
+    end
+  end
+  
+  async:Do(function()
```

Comment:
Since we last spoke, I added `async.Each` and `async.Batch`. While not required, for this workflow, you may want to look at either Each or Batch.